### ISSUE TITLE
Biomarker extraction

### ISSUE THREAD

**Issue by xiaoyanzhao75** [xiaoyanzhao75/Share-with-Ou#1]
Use data science method of your choice to extract relevant clinical variables below
Report Date

(specimen) Collection Date

Biomarker and [Result

![Image](https://github.com/user-attachments/assets/bfa95877-93ed-4f6d-a45a-27c72b47d834)
![Image](https://github.com/user-attachments/assets/c93971f2-2d9c-471a-b862-20e9c2ca35cb)
![Image](https://github.com/user-attachments/assets/e0e0abd8-3c7d-436a-a7de-6e98ffe11634)

**Comment by xiaoyanzhao75** [comment]
[File 1. Biomarker_Data_with_CPT_Codes_and_Descriptions.csv](https://github.com/user-attachments/files/16908805/File.1.Biomarker_Data_with_CPT_Codes_and_Descriptions.csv)
[File 2 Biomarker_Data_with_CPT_Codes_and_Descriptions.csv](https://github.com/user-attachments/files/16908807/File.2.Biomarker_Data_with_CPT_Codes_and_Descriptions.csv)
[File 3. Biomarker_Data_with_CPT_Codes_and_Descriptions.csv](https://github.com/user-attachments/files/16908806/File.3.Biomarker_Data_with_CPT_Codes_and_Descriptions.csv)

**Comment by xiaoyanzhao75** [comment]
The results are shown as above. Code that I used in Chatgpt: **use data science method to extract: report date, collection date, biomarker and result. Map biomarkers with possible CPT codes and description. Arrange them in one table. if report date or collection date are unknown, populate NA**

As a Clinical Informaticist with a PhD in Immunology and Genetics and a Healthcare Analyst, I am proud to see the growing application of clinical genetic testing in research, particularly in cancer. I have also included possible CPT codes associated with these tests to support future analytics and enhance data-driven insights.

I also would like to add a few review papers regarding the application of cancer genetic tests in real world data like medical claims or EHR, as well as CMS medical coding guidelines. 

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10765765/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7000322/

Genetic coding guideline from CMS: https://www.cms.gov/medicare-coverage-database/view/article.aspx?articleid=58917
